joomla/joomla-cms
# [4.0] Fancy Select custom element improvements

### Summary of Changes

- Throw an error if the Joomla API hasn't been initiated
- Move initial checks and some variables to constructor
- Use template literals
- `for` > `forEach`
- eslint fixes

### Testing Instructions

1. Go to create a new article
2. Check the category selection dropdown still works as expected
